### ISSUE TITLE
AWS SDK conflict

### ISSUE THREAD

**Issue by TheUnknownJG** [deliciousbrains/wp-amazon-s3-and-cloudfront#318]
I am attempting to write a piece into our site that uses the AWS SDK (using their aws.phar v3.15.0 file) so I can do an SNS push to our mobile devices.  I had a stand-alone test page written that works perfectly.  When I attempted to include this into the WP site, I started getting repeated errors in the error logs because the SDK I was including was conflicting with what is being pulled in from your Amazon Web Service plugin.

The site also threw an error message on the settings page of the Offload S3 plugin (again, because the libraries conflicted).  Screenshot of the error is attached.

What I would like to know is, is there a way to call upon the libraries you are already pulling in?  If I could somehow use those libraries instead of having to include the SDK that would solve all my problems.  Of course, I also don't know if your AWS plugin contains the full set of libraries from Amazon, or if you have just selectively included the libraries necessary for your plugin to function.

Any help from anyone in the forum is greatly appreciated!!  Thnx!

![offload s3 error](https://cloud.githubusercontent.com/assets/6464750/14745155/fb2c7a32-0877-11e6-8e7f-50c9e064088e.PNG)

**Comment by TheUnknownJG** [comment]
Nevermind.  I finally found another post referencing this exact issue.  I will leave this here for others to find as well though.

The problem was caused by the fact that the Offload S3 plugin does not play well with the AWS SDK V3.  Going back to version 2 (v2.8.28 being the last v2 version) of the SDK fixed my problems.

Here is a link to legacy version of the SDK for your convenience.
https://github.com/aws/aws-sdk-php/releases
